object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -173,7 +172,7 @@ boolean returnsThis(final MethodInvocationTree tree) {
 
   /**
    * Continue to trust but not check the old {@link
-   * org.checkerframework.checker.builder.qual.ReturnsReceiver} annotation, for
+   * org.checkerframework.checker.returnsreceiver.builder.qual.ReturnsReceiver} annotation, for
```

Comment:
This annotation's package cannot change. It's generated by Lombok, so we have to continue to use the old fully-qualified name here. Please restore it; it can be part of object-construction-qual if necessary.